lib/auth/style.scss

```diff
@@ -194,4 +212,21 @@
   svg.icon-mail {
     color: $studio-gray-30;
   }
+  input {
+    border-color: $studio-gray-60;
+
+    &:focus {
+      box-shadow: 0 0 0 2px $studio-simplenote-blue-30;
```

Comment:
I think we have to add here `border-color: transparent;`, otherwise the dark theme sets the border color to grey when it's focused as per line 216.

Here is how it looks like currently, observe the thin inner grey line:
<img width="1192" alt="Screenshot 2021-04-13 at 11 28 16" src="https://user-images.githubusercontent.com/14905380/114530769-8d2dc780-9c4b-11eb-8196-1aa987746533.png">

And here are the styles of the email input when it's focused, observe that `.theme-dark .login input` sets the border color to grey:
<img width="313" alt="Screenshot 2021-04-13 at 11 29 34" src="https://user-images.githubusercontent.com/14905380/114530816-96b72f80-9c4b-11eb-8110-576d2bc66269.png">

Reply:
Thanks for catching this. 
I think I'll solve this by going with the recommendation in your other comment to use a combination of border and box-shadow.